app/uk/gov/hmrc/tariffclassificationfrontend/views/components/input_autoComplete.scala.html

```diff
@@ -0,0 +1,172 @@
+@*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import play.api.libs.json.Json
+
+@(
+        field: Field,
+        label: String,
+        inputClass: Option[String] = None,
+        hint: Option[String] = None,
+        labelClass: Option[String] = None,
+        autoCompleteOptions: Seq[String]
+)(implicit messages: Messages)
+
+<div class="form-group @if(field.hasErrors){form-group-error}">
+    <label class="form-label" for="@{field.id}">
+        <span class="bold @if(labelClass.nonEmpty){@labelClass}">@label</span>
+        @if(hint.nonEmpty){
+            <span class="form-hint">@hint</span>
+        }
+        @field.errors.map { error =>
+            <span class="error-message" id="error-message-@{field.id}-input">@messages(error.message, error.args: _*)</span>
+        }
+    </label>
+    <div class="form-control-wrapper autocomplete w-100">
+        <input
+        class="form-control @inputClass"
+        type="text"
+        id="@{field.id}"
+        name="@{field.name}"
+            @if(field.hasErrors){aria-describedby="error-message-@{field.id}-input"}
+        />
+    </div>
+</div>
+
+<script>
+
+        /*Polyfill for includes which doesn't work in IE 11*/
+        if (!String.prototype.includes) {
```

Comment:
Are we happy to have this JavaScript block here?
I wonder if it would be more readable to move inside the `app/assets/javascripts` folder

Reply:
Moved it as suggested